task_processing/plugins/kubernetes/kubernetes_pod_executor.py

```diff
@@ -66,8 +69,32 @@ def run(self, task_config: KubernetesTaskConfig) -> None:
                     ),
                 ),
             )
+        # TODO (TASKPROC-238): Add volume, cpu, gpu, desk, mem, etc.
+        container = V1Container(
+            image=task_config.image,
+            name=task_config.name,
+            command=[task_config.command]
+        )
+        pod = V1Pod(
+            metadata=V1ObjectMeta(
+                name=task_config.pod_name,
+                namespace=self.namespace
+            ),
+            spec=V1PodSpec(
+                restart_policy=task_config.restart_policy,
+                containers=[container]
+            ),
+        )
+
+        try:
+            self.kube_client.core.create_namespaced_pod(namespace=self.namespace, body=pod)
+        except Exception as e:
+            logger.error(f"Failed to create pod {task_config.pod_name}: {e}")
```

Comment:
super-nit: we should probably be consistent and use `logger.exception` here since that's what we're using elsewhere (that'll also automatically include the traceback below the log message
```suggestion
        except Exception:
            logger.exception(f"Failed to create pod {task_config.pod_name}")
```